internal/k8s/ako_init.go

```diff
@@ -420,6 +421,7 @@ func (c *AviController) FullSyncK8s() {
 			utils.AviLog.Errorf("Unable to retrieve the gateways during full sync: %s", err)
 		} else {
 			for _, gatewayObj := range gatewayObjs {
+				status.InitializeGatewayConditions(gatewayObj)
```

Comment:
Will this put a working `synced` gateway into initialising state?

Reply:
added a check in the initialisation function, it shouldn't reinitialise things anymore. thanks